CMakeLists.txt

```diff
@@ -769,17 +769,21 @@ endif()
 option(PREFER_EXTERNAL_LIBS "Tries to use system libs where possible." ON)
 
 macro(prefer_package LIB_NAME LIB_CMAKE)
-    if (PREFER_EXTERNAL_LIBS AND NOT NACL)
-        find_package(${LIB_NAME})
+	if (NOT ${LIB_NAME}_FOUND)
+		if (PREFER_EXTERNAL_LIBS AND NOT NACL)
```

Comment:
The rest of this file is using spaces, not tabs

Reply:
Fixed.